src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -56,36 +55,66 @@ public void isParallelTesting_ReturnsParallelTesting() {
     }
 
     @Test
-    public void getJenkinsBasePath_OnSampleBath_ReturnsJenkinsPath() {
+    public void getJenkinsBuildURL_OnSampleURL_ReturnsJenkinsBuildURL() {
 
         // give
         Configuration configuration = new Configuration(outputDirectory, projectName);
-        String basePath = "abc321";
-        configuration.setJenkinsBasePath(basePath);
+        String jobPath = "http://localhost:8080/jobs/TEST/2/";
+        configuration.setJenkinsBuildURL(jobPath);
 
         // when
-        String path = configuration.getJenkinsBasePath();
+        String path = configuration.getJenkinsBuildURL();
 
         // then
-        assertThat(path).isEqualTo(basePath);
+        assertThat(path).isEqualTo(jobPath);
     }
 
     @Test
-    public void getJenkinsBasePath_OnEmptyPath_ReturnsJenkinsPath() {
+    public void getJenkinsPreviousBuildURL_OnSampleURL_ReturnsJenkinsPreviousBuildURL() {
 
         // give
         Configuration configuration = new Configuration(outputDirectory, projectName);
-        String basePath = StringUtils.EMPTY;
-        configuration.setJenkinsBasePath(basePath);
+        String jobPath = "http://localhost:8080/jobs/FOLDER/jobs/TEST/1/";
+        configuration.setJenkinsPreviousBuildURL(jobPath);
 
         // when
-        String path = configuration.getJenkinsBasePath();
+        String path = configuration.getJenkinsPreviousBuildURL();
 
         // then
-        assertThat(path).isEqualTo("/");
+        assertThat(path).isEqualTo(jobPath);
     }
 
     @Test
+    public void getJenkinsNextBuildURL_OnSampleURL_ReturnsJenkinsNextBuildURL() {
+
+        // give
+        Configuration configuration = new Configuration(outputDirectory, projectName);
+        String jobPath = "http://localhost:8080/jobs/TEST/3/";
```

Comment:
Build numbers aren't actually guaranteed to be continuous (Although they'll always increase). Either by manually editing the nextBuildNumber file or using the Set Next Build Number plugin build numbers can jump up by any interval. Which means you can't rely on the previous build being current--.